third_party/blink/renderer/platform/fonts/font_fallback_list.cc

```diff
@@ -107,22 +107,44 @@ const SimpleFontData* FontFallbackList::DeterminePrimarySimpleFontDataCore(
     const FontDescription& font_description) {
   bool should_load_custom_font = true;
 
+  recordreplay::Assert(
+    "[RUN-1219-1597] FontFallbackList::DeterminePrimarySimpleFontDataCore #0"
+  );
   for (unsigned font_index = 0;; ++font_index) {
+    recordreplay::Assert(
+      "[RUN-1219-1597] FontFallbackList::DeterminePrimarySimpleFontDataCore #1 fontidx=%u",
+      font_index
+    );
     const FontData* font_data = FontDataAt(font_description, font_index);
     if (!font_data) {
       // All fonts are custom fonts and are loading. Return the first FontData.
       font_data = FontDataAt(font_description, 0);
-      if (font_data)
+      if (font_data) {
+        recordreplay::Assert(
+          "[RUN-1219-1597] FontFallbackList::DeterminePrimarySimpleFontDataCore #2 fontidx=%u gotFontData",
```

Comment:
I think there's room for different styles here, this will have a negligible impact on the recording size and makes it easier to read what is mismatched.